src/commcare_cloud/ansible/roles/nginx/vars/cas_ssl.yml

```diff
@@ -61,7 +61,38 @@ nginx_sites:
         # If the header is not provided, do not rate limit
         - key: '""'
           value: '""'
+    - name: "$restore_limit"
+      variable: "$days_since_last_sync"
+      var_mapping:
+        - key: "0"
+          value: 0
+        - key: "~\d+"
+          value: 1
+        - key: "default"
+          value: '""'  
+    - name: "$synced_today_bucket"
+      variable: "$restore_limit"
+      var_mapping:
+        - key: "0"
+          value: "$limit_bucket"
+        - key: "default"
+          value: '""'
+    - name: "$synced_old_bucket"
+      variable: "$restore_limit"
+      var_mapping:
+        - key: "1"
+          value: "$limit_bucket"
+        - key: "default"
+          value: '""'        
+
    limit_req_zones:
+    - name: "synced_today_bucket"
+      zone: "$restore_limit"
+      size: "10m"
+      rate: "40r/s"
+    - name: "synced_old_bucket"
+      zone: "$restore_limit"
+      rate: "60r/s"  
     - name: "incremental_restore" # Limits restores that are not new device or 412s
```

Comment:
this can be removed completely since both `synced_today_bucket` and `synced_old_bucket` are 'incremental' syncs

Reply:
Removed